include/beman/inplace_vector/inplace_vector.hpp

```diff
@@ -271,6 +271,13 @@ Software.
 // Artifact from previous implementation, can be used as hints for optimizer
 #define IV_EXPECT(EXPR)
 
+#if (!defined(__EXCEPTIONS) && !defined(_CPPUNWIND)) ||                        \
+    __STDC_HOSTED__ == 0 || BEMAN_IV_FREESTANDING_FORCE_TEST
```

Comment:
```suggestion
    __STDC_HOSTED__ == 0 || BEMAN_IV_FREESTANDING
```
I'd suggest just making this a build option that the user can pass and explain the rules for when the implementation becomes freestanding in the readme.md

Reply:
Use `BEMAN_INPLACE_VECTOR_FREESTANDING` for consistent naming.